recipe/meta.yaml

```diff
@@ -9,10 +9,11 @@ source:
   url: https://github.com/PierreRaybaut/guidata/archive/v{{ version }}.tar.gz
   sha256: 2ffc8e6dd27a57c6ec0e260986368beeddbd2fa5588f5200614b307b27a39e37
   patches:
-   - removed_entry_points.patch
+    - removed_entry_points.patch
```

Comment:
Explanation why this patch is needed

Reply:
The first one I don't know the reason

Reply:
If I do remember well, this patch is not necessary and you have to add the entry points in this recipe

Please see
https://docs.conda.io/projects/conda-build/en/latest/source/resources/define-metadata.html#python-entry-points

Reply:
Ohhh cool